projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -348,6 +348,21 @@ public function configure() {
 			'stc_enabled'
 		);
 
+		/** Enable Subscribe Modal */
+
+		add_settings_field(
```

Comment:
I think this should also be hidden behind the feature flag.

Reply:
[done](https://github.com/Automattic/jetpack/commit/41c5af8c58e3698130b7906748bdacffcb8ccbdb), where these show up?